mmaction/datasets/pipelines/formating.py

```diff
@@ -174,16 +205,20 @@ class Collect:
                 - mean - per channel mean subtraction
                 - std - per channel std divisor
                 - to_rgb - bool indicating if bgr was converted to rgb
+        nested (bool): If set as True, will apply data[x] = [data[x]] to all
+            items in data. The arg is added for compatibility. Default: False.
     """
 
     def __init__(self,
                  keys,
                  meta_keys=('filename', 'label', 'original_shape', 'img_shape',
                             'pad_shape', 'flip_direction', 'img_norm_cfg'),
-                 meta_name='img_meta'):
+                 meta_name='img_metas',
```

Comment:
Reason to use plural form?

`metas` feels (personal feeling) unnatural.

Reply:
To be compatible with mmdet:
In mmdet the arg is named img_metas, and I use the detector in mmdet.

Reply:
okay...

Reply:
Will this change affect other methods?

Reply:
Should be OK, I have changed everywhere to img_metas.